src/components/Search.js

```diff
@@ -54,9 +59,10 @@ export const Search = () => {
   useSoftkey('Search', {
     right: i18n.i18n('softkey-settings'),
     onKeyRight: () => { window.location.hash = '/settings' },
-    center: current.type === 'DIV' ? i18n.i18n('centerkey-select') : '',
+    center: current.type === 'DIV' ? i18n.i18n('centerkey-select') : i18n.i18n('softkey-search'),
```

Comment:
[Waiting for confirmation](https://phabricator.wikimedia.org/T248080#5988593) on whether pressing this softkey should make something happen

Reply:
Confirming there's no action attached to it